HPXMLtoOpenStudio/resources/airflow.rb

```diff
@@ -439,7 +441,11 @@ def self.apply_natural_ventilation_and_whole_house_fan(model, site, vent_fans_wh
     vent_program.addLine("Set #{whf_flow_actuator.name} = 0") # Init
     vent_program.addLine("Set #{liv_to_zone_flow_rate_actuator.name} = 0") unless whf_zone.nil? # Init
     vent_program.addLine("Set #{whf_elec_actuator.name} = 0") # Init
-    vent_program.addLine('If (Wout < MaxHR) && (Phiout < MaxRH) && (Tin > Tout) && (Tin > Tnvsp) && (ClgSsnAvail > 0)')
+    if @outage_sensor.nil?
+      vent_program.addLine('If (Wout < MaxHR) && (Phiout < MaxRH) && (Tin > Tout) && (Tin > Tnvsp) && (ClgSsnAvail > 0)')
+    else
+      vent_program.addLine("If ((Wout < MaxHR) && (Phiout < MaxRH) && (Tin > Tout) && (Tin > Tnvsp) && (ClgSsnAvail > 0)) || ((Tin > Tout) && (Tin > Tnvsp) && (#{@outage_sensor.name} == 0))") # 0 means during the power outage period
+    end
```

Comment:
Why are you having to do this? I thought you were handling the natural ventilation availability in schedule.rb.

Also, why does a zero mean during an outage period? That is an incredible confusing convention and it doesn't match your documentation.

Reply:
On the first, @jmaguire1 would be able to provide an explanation.

On the second, we're talking about the outage _sensor_ here, not the detailed outage schedule itself. I create the outage sensor [here](https://github.com/NREL/OpenStudio-HPXML/blob/power-outage2/HPXMLtoOpenStudio/measure.rb#L279-L291) (regardless if using detailed schedules or not). It uses the ScheduleConstant class and passes in `off_periods` which assigns the 0s during the outage period. So for this case 1 means no outage and 0 means outage.

Reply:
Then it should be called an `off_period_sensor` (or similar), not `outage_sensor`.

Reply:
Err, wait, I'm still confused. If 1 means no outage, then it is not an off period, nor an outage period. Either the convention or variable names need to be corrected.

Reply:
Sounds like it's really an `availability_sensor`.

Reply:
@shorowit: We are using the availability schedule, but we also constrain the window opening based on temperatures and humidity. We're assuming that if it's an outage, you'd ignore the humidity constraints we normally put on window opening per the old HSP guidance (RH < 70% and w < 0.015). I looked with Phil at one of his runs and the humidity constraints were preventing the window from opening during the entire outage period even though the sensible cooling would have really helped.

Reply:
Got it, that makes sense. Thanks for explaining @jmaguire1. I would suggest a comment in the code so that it's more clear what is intended.